src/index.html

```diff
@@ -5,8 +5,372 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo logo">
+        <img
+          src="images/logo.png"
+          class="logo__img"
+          alt="Logo Image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"> Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="product-container main__product-container">
+        <div class="card product-container__card" data-qa="card">
+          <div class="card__content">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac Image"
+              class="card__image">
+
+            <div class="card__header">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+
+            <div class="card__code">
+              Product code: 195434
+            </div>
+            <div class="card__review review">
+              <div class="stars stars--4 review__stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="review__text">Reviews: 5</div>
+            </div>
+
+            <div class="card__price">
+              <p class="card__price-text">Price:</p>
+              <p class="card__price-dolars">$2,199</p>
+            </div>
+
+            <a
+                href="#"
+                class="card__buy buy"
+                data-qa="hover"
```

Comment:
when you look at test results telling `waiting for selector "[data-qa="card-hover"]"`
you could check the [task description](https://github.com/huberciak/layout_catalog-en/tree/develop#:~:text=add%20data%2Dqa%3D%22card%2Dhover%22)
> add data-qa="card-hover" (not just hover) to the link Buy inside the first card

that might help you with tests